first-analysis-steps/eos-storage.md

```diff
@@ -56,178 +59,51 @@ thanks to the `accessURL` function which will give you the URL of your output
-{% endcallout %} 
-
-The final line tells your `ganga` job that the `outputfiles` of this
-job that need special treatment.
+There are two useful workspaces on EOS:
+- `/eos/user` which is your private EOS space, also accessible via [CERNBox](https://cernbox.cern.ch/), and has 1TB of space;
+- `/eos/lhcb` which is the LHCb area and contains the following sub-directories:
+  - `/eos/lhcb/user` which is your user area, where you can store your tuples, and has 2TB of space;
+  - `/eos/lhcb/wg` which contains the Working Group areas (e.g. for preserving tuples for completed analyses);
+  - `/eos/lhcb/grid/` which is the path to the CERN grid site; and many others.
 
-One important thing to note is that `ganga` has to be running after
-your job has completed to copy the files to EOS. The job can not
-copy things to EOS itself. You can leave `ganga` running in a
-screen session and it will copy files as they
-become available.
-
-Once your job has completed and the files have been copied to
-EOS by `ganga` you can access them from your terminal by mounting
-your EOS area.
-
-{% callout "Important!" %}
-Important: mounting eos is discouraged by the LHCb data management team,
-and no support will be provided by them for problems/questions related to it.
-{% endcallout %} 
-
-On `lxplus` EOS is mount under /eos like this:
+Let's inspect your user area:
 
 ```bash
-$ ls -al /eos/user
-```
-
-```output
-total 0
-drwxr-xr-x. 1 daemon root 269935142640435 Oct  6  2014 .
-drwxr-xr-x. 9 root   root               0 Oct 24 11:30 ..
-drwxr-xr-x. 1 daemon root               0 Dec 15  2014 0
-drwxr-xr-x. 1 daemon root  15407422735850 Oct 28 10:13 a
-drwxr-xr-x. 1 daemon root   4213907467001 Oct 26 16:10 b
-drwxr-xr-x. 1 daemon root   4592926796516 Oct 27 21:49 c
-drwxr-xr-x. 1 daemon root   3951569840852 Oct 26 08:20 d
-drwxr-xr-x. 1 daemon root   3104338965290 Oct 27 14:27 e
-drwxr-xr-x. 1 daemon root   2243039535809 Oct 27 13:48 f
-drwxr-xr-x. 1 daemon root   5916063359888 Oct 28 10:05 g
-drwxr-xr-x. 1 daemon root   2778840054706 Oct 30 19:56 h
-drwxr-xr-x. 1 daemon root    442304842811 Oct 26 10:35 i
-drwxr-xr-x. 1 daemon root   8360665587286 Oct 30 13:46 j
-```
-
-If you list the contents of the `/eos` directory you should see
-
-```output
-lhcb ship user
+$ ls -al /eos/lhcb/user/a/another
 ```
 
-If you used `/eos/lhcb/user/a/another/starterkit/{jid}_{fname}` when
-configuring your `MassStorageFiles` there should now be files
-visible here:
+Files stored on EOS can be open in `ROOT` the usual way:
 
 ```bash
-$ ls /eos/lhcb/user/a/another/starterkit
+$ root /eos/lhcb/user/a/another/myfavouritefile.root
 ```
 
-Once you have found your file you can open it in `ROOT` like this:
+You may also use the following path which is independent of the EOS mount (the mount is in general not very reliable):
 
 ```bash
-$ root /eos/lhcb/user/a/another/starterkit/myfavouritefile.root
+$ root root://eoslhcb.cern.ch//eos/lhcb/user/a/another/myfavouritefile.root
```

Comment:
The prefix is available as the `EOS_MGM_URL`, so we could also add an equivalent line:

```
$ root $EOS_MGM_URL//eos/lhcb/user/a/another/myfavouritefile.root
```

or maybe just a parenthetical.

Reply:
Learning something new this evening, thanks!

Reply:
I'd suggest not using `$EOS_MGM_URL` so they can also run it locally. (It might also be a mess for people who are part of multiple experiments.)

Reply:
Also I think new users aren't given space on `root://eoslhcb.cern.ch//eos/lhcb/user` any more and are encouraged to use `root://eosuser.cern.ch//eos/user/` instead.

Reply:
I didn't know about not getting anything in `eoslhcb`. That seems like an important point. So new users only get the 1 TB of space on `eosuser` now?

Reply:
The person who created lhcb account in April this year has the /eos/lhcb/user space

Reply:
I cannot find more recent ones now

Reply:
I've found someone who made there account in October and they have eos lhcb space so I must be wrong.

Reply:
Okay, if you wish I revert back the `EOS_MGM_URL` part, and we leave it like that?